src/module.c

```diff
@@ -3563,8 +3557,16 @@ static int initSearchCluster(RedisModuleCtx *ctx, RedisModuleString **argv, int
     num_connections_per_shard = RSGlobalConfig.numWorkerThreads + 1;
   }
 
-  MRCluster *cl = MR_NewCluster(NULL, num_connections_per_shard);
-  MR_Init(cl, clusterConfig.timeoutMS);
+  size_t num_io_threads;
+  if (clusterConfig.coordinatorIOThreads) {
+    num_io_threads = clusterConfig.coordinatorIOThreads;
+  } else {
+    num_io_threads = RSGlobalConfig.numWorkerThreads + 1;
```

Comment:
Is this relevant? `coordinatorIOThreads` cannot be 0, right? Defaults to 1, and can be set to at least 1